out-of-sync-global-dns/workload.yaml

```diff
@@ -0,0 +1,36 @@
+---
```

Comment:
Add a service account and RBAC to allow the pod to access the needed resources

Reply:
Fixed.
serviceaccount/detect-orphaned-machines created
clusterrole.rbac.authorization.k8s.io/detect-orphaned-machines created
clusterrolebinding.rbac.authorization.k8s.io/detect-orphaned-machines created
deployment.apps/detect-orphaned-machines configured